cwl/src/main/scala/cwl/CommandOutputBinding.scala

```diff
@@ -39,9 +39,10 @@ case class CommandOutputBinding(
     val womMapType = WomMapType(WomStringType, WomStringType)
     val womMaps = paths map { path =>
       // TODO: WOM: basename/dirname/size/checksum/etc.
+      val globPathWithDirectory = GlobFunctions.prefixWithGlobDir(path)
 
       val contents: Map[WomValue, WomValue] =
-        if (loadContents) Map(WomString("contents") -> WomString(load64KiB(path, ioFunctionSet))) else Map.empty
+        if (loadContents) Map(WomString("contents") -> WomString(load64KiB(globPathWithDirectory, ioFunctionSet))) else Map.empty
```

Comment:
Make sure to prefix the filename with the glob directory. This was the "accidental success" on the shared filesystem where this code was picking up the version of the stdout file that it shouldn't have seen.